pallets/parachain-staking/src/types.rs

```diff
@@ -219,8 +246,8 @@ where
 		}
 	}
 
-	// Returns Some(remaining balance), must be more than MinDelegatorStk
-	// Returns None if delegation not found
+	/// Returns Some(remaining stake for delecator), must be more than
+	/// MinDelegatorStk Returns None if delegation not found
 	pub fn rm_delegation(&mut self, collator: AccountId) -> Option<Balance> {
```

Comment:
Maybe collator could be directly a `&AccountId`, so we don't have to clone everytime just to use it as a reference in the search?

Reply:
very good point!